src/ethereum/ethEngine.js

```diff
@@ -949,6 +949,8 @@ export class EthereumEngine extends CurrencyEngine<EthereumPlugin> {
       otherParams // otherParams
     }
 
+    this.warn(`makeSpend\n${cleanTxLogs(edgeTransaction)}`)
```

Comment:
This could get noisy and lag the system as each keystroke in the FlipInput would call makeSpend. I'd request removing this.